File: Assignment_24/program1.c
////////////////////////////////////////////////////////////////////
//
// Name :          program1.c
// Discription :   Check Capital Character 
// Input :         String
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          07/06/2025
//
////////////////////////////////////////////////////////////////////

/*
  Start
    Accept String from user 
      Check capital character form String
  Stop    
*/

#include<stdio.h>

int CountCapital(char *str)
{
  int iCnt = 0, iCount = 0;

  if(NULL == str)
  {
     return -1;
  }

  while(*str != '\0')
  {
     if((*str >= 'A') && (*str <= 'Z'))
     {
        iCount++;
     }
     str++;
  }

  return iCount;
}

int main()
{
   char Arr[20];
   int iRet = 0;

   printf("Enter string :\n");
   scanf("%[^'\n']s",Arr);

   iRet = CountCapital(Arr); 

   printf("%d",iRet);

   return 0;

}

File: Assignment_55/program1.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program1.java
// Discription :   use to replace capital character with small character
// Input :         Integer
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          19/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray
{
    public void ReplaceArray(char Arr[])
    {
        int i = 0;
    
        for(i = 0; i < Arr.length; i++)
        {
           if(Arr[i] >= 'A' && Arr[i] <= 'Z')
           {
                Arr[i] += 32;
           }
        }

        for(i = 0; i < Arr.length; i++)
        {
           System.out.print(Arr[i]+"\t");
        }
        
    }
}

class program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int i = 0;

        System.out.println("Enter Array of Characters : ");
        String str = sobj.nextLine();

        char Arr[] = str.toCharArray();

        MyArray mobj = new MyArray();

        mobj.ReplaceArray(Arr);

        sobj.close();
    }
}